be/src/util/slice.h

```diff
@@ -168,8 +168,8 @@ struct Slice {
     bool trim_quote() {
         int32_t begin = 0;
         bool change = false;
-        if (size > 2 && ((data[begin] == '"' && data[size - 1] == '"') ||
-                         (data[begin] == '\'' && data[size - 1] == '\''))) {
+        if (size >= 2 && ((data[begin] == '"' && data[size - 1] == '"') ||
```

Comment:
Method trim_double_quotes in slice class also need the change.

Reply:
OK